packages/bsp/common/usr/lib/armbian/armbian-firstlogin

```diff
@@ -270,58 +302,65 @@ if [[ -f /root/.not_logged_in_yet && -n $(tty) ]]; then
 	desktop_lightdm=$(dpkg-query -W -f='${db:Status-Abbrev}\n' lightdm 2>/dev/null)
 	desktop_gdm3=$(dpkg-query -W -f='${db:Status-Abbrev}\n' gdm3 2>/dev/null)
 
-	echo -e "\nWaiting for system to finish booting ...\n"
+	echo -e "\nWaiting for system to finish booting ..."
 
 	systemctl is-system-running --wait >/dev/null
 
-	if [ "$IMAGE_TYPE" != "nightly" ]; then
-		if [ "$BRANCH" == "dev" ]; then
-			echo -e "\nYou are using an Armbian preview build !!!"
-			echo -e "\nThis image is provided \e[0;31mAS IS\x1B[0m with \e[0;31mNO WARRANTY\x1B[0m and \e[0;31mNO END USER SUPPORT!\x1B[0m.\n"
-		elif [ "$DISTRIBUTION_STATUS" != "supported" ]; then
-			echo -e "\nYou are using an Armbian with unsupported ($DISTRIBUTION_CODENAME) userspace !!!"
-			echo -e "\nThis image is provided \e[0;31mAS IS\x1B[0m with \e[0;31mNO WARRANTY\x1B[0m and \e[0;31mNO END USER SUPPORT!\x1B[0m.\n"
-		fi
-	else
-		echo -e "\nYou are using an Armbian nightly build meant only for developers to provide"
-		echo -e "constructive feedback to improve build system, OS settings or user experience."
-		echo -e "If this does not apply to you, \e[0;31mSTOP NOW!\x1B[0m. Especially don't use this image for"
-		echo -e "daily work since things might not work as expected or at all and may break"
-		echo -e "anytime with next update. \e[0;31mYOU HAVE BEEN WARNED!\x1B[0m"
-		echo -e "\nThis image is provided \e[0;31mAS IS\x1B[0m with \e[0;31mNO WARRANTY\x1B[0m and \e[0;31mNO END USER SUPPORT!\x1B[0m.\n"
-	fi
-
-	echo -e "New to Armbian? Documentation: \e[1m\e[39mhttps://docs.armbian.com\x1B[0m Support: \e[1m\e[39mhttps://forum.armbian.com\x1B[0m\n"
+	clear
+	TERM=linux toilet -f banner  "A R M B I A N"
```

Comment:
I would suggest in order to increase accessibility for screen readers etc. to avoid ascii art and use "bold" or other console formatting instead (although I can understand that you might want to have this visibility on screenshots etc.)